automarker_2/automarker_app/management/commands/check_all_configuration.py

```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+from automarker_app.lib.marker import get_all_marker_configs
+from automarker_app.lib.constants import CONFIG_STATUS_DEBUG, CONFIG_STATUS_PRODUCTION
+
+
+class Command(BaseCommand):
```

Comment:
Isn't this supposed to like actually run the configs like what "check_project_configuration" does?

[Reference](https://github.com/Umuzi-org/Tilde/blob/8b2540b663f25e3cfbe7f0c84dcb9c5bd655209f/automarker_2/automarker_app/management/commands/check_project_configuration.py#L11)

Reply:
I agree when looking at the script names but the description made me think that this should look more like the print_configuration script :shrug: